_episodes/02-practice-learning.md

```diff
@@ -8,8 +8,11 @@ questions:
 - "Who is a typical Carpentries learner?"
 - "How can we help novices become competent practitioners?"
 objectives:
+- "Compare and contrast the three stages of skill acquisition."
+- "Identify a mental model and explain what it represents." 
 - "Explain why practice and feedback are important for learning."
-- "Use formative assessments to guide your teaching."
+- "Explain the purpose of formative assessments."
+- "Give an example of a formative assessment and when you would use it." 
```

Comment:
I *like* this objective, but I think that this episode functions more at the level of identifying formative assessments in past experiences, with some application specifically for identifying misconceptions with MCQ. Generating formative assessments generally happens in `15-lesson-study`.

Reply:
I was building off of this exercise: Based on your previous educational experience (or even this training so far!) what types of formative assessments do you know about?

So I do think that participants should be able to give an example of a formative assessment, beyond MCQs. Maybe take off the "and when you would use it"?

Reply:
That makes sense, thinking a little more, I think the issue is that on my first reading, "give an example" evoked the create level of bloom's but this episode I think covers more the understand level.  Maybe something like this?

```suggestion
- "Identify formative assessments in your teaching and learning experiences." 
```

Reply:
I'm not sure that quite captures what I'm going for which is if someone stopped you on the street, you could give them an example, but I do think it captures what happens in the lesson, so let's go with this for now.